util/bitstream.go

```diff
@@ -226,3 +256,46 @@ func ParseUInt16(data []byte, bitStartIndex uint16) (uint16, error) {
 	}
 	return binary.BigEndian.Uint16([]byte{leftByte, rightByte}), nil
 }
+
+func (bs *BitStream) ReadTwoBitField(numFields int, err error) ([]byte, error) {
```

Comment:
We may want to define the return value as its own type, so we can attach a helper function to it to decompose the two bits into the two booleans that those two bits correspond to. Might also allow us to label the indices in some way so that the user of the library doesn't have to reference the original spec too closely.